syncstorage-postgres/src/db/db_impl.rs

```diff
@@ -135,14 +136,41 @@ impl Db for PgDb {
     }
 
     async fn get_bsos(&mut self, params: params::GetBsos) -> Result<results::GetBsos, Self::Error> {
-        todo!()
+        let selection = (
+            bsos::bso_id,
+            bsos::modified,
+            bsos::payload,
+            bsos::sortindex,
+            bsos::expiry,
+        );
+        let (bsos, next_offset) = bsos_query!(self, params, selection, GetBso);
+        let items = bsos
+            .into_iter()
+            .map(|bso| results::GetBso {
+                id: bso.bso_id,
+                modified: SyncTimestamp::from_milliseconds(
+                    bso.modified.and_utc().timestamp_millis() as u64,
+                ),
```

Comment:
We can cleanup these to/from conversions later, especially since we're still discussing use of `TIMESTAMP` types in the other PR.